generated/nodejs/google-cloud-node/packages/trace/test/trace.js

```diff
@@ -0,0 +1,60 @@
+var assert = require('assert');
+var traceV1 = require('../src/').v1();
+
+describe('Trace', function() {
+  // Simple method demo
+  describe('patch', function() {
+    it('Test patch tree', function(done) {
+      var api = traceV1.traceServiceApi();
+
+      // Initialization
+      var trace = {projectId: 'test-project',
+                   traceId: 'test-id',
+                   spans: []};
+      var traces = [trace];
+      var projectId = 'test-project';
+
+      // Mock Grpc layer
+      api._patchTraces = function(req, options, callback) {
+        assert.equal(req.traces.length, 1);
+        assert.equal(req.traces[0], trace);
+        assert.equal(req.projectId, projectId);
+        callback(null);
+      }
+
+      api.patchTraces('test-project', traces, function(err) {
```

Comment:
It seems trace client hasn't been regenerated for a while. Let me do that first -- the API surface has change significantly.